responsibleai/responsibleai/managers/error_analysis_manager.py

```diff
@@ -165,28 +152,11 @@ def from_json(json_str):
 
 
 class ErrorAnalysisManager(BaseManager):
-
-    """Defines the ErrorAnalysisManager for discovering errors in a model.
```

Comment:
This change feels like it should be a different PR....

Reply:
Good point. I am working on that right now and will do all unrelated doc changes there. Stay tuned!

Reply:
#1209 exists now! Once it's merged the changes should vanish from here :-)